src/Mailer/Transport/DebugKitTransport.php

```diff
@@ -0,0 +1,84 @@
+<?php
+namespace DebugKit\Mailer\Transport;
+
+use Cake\Core\App;
+use Cake\Mailer\AbstractTransport;
+use Cake\Mailer\Email;
+
+/**
+ * Debug Transport class, useful for emulating the email sending process and inspecting
+ * the resultant email message before actually sending it during development
```

Comment:
resultant -> resulting? Not sure what's more corect, but the latter sounds simpler to me, especially for non-English native people.